scripts/publish-docs.sh

```diff
@@ -14,7 +14,7 @@ if ! [[ "${VERSION}" =~ ^([0-9]+\.){2}[0-9]+(-.+)?$ ]]; then
   exit 1
 fi
 
-
+git checkout -- .
```

Comment:
Would it make sense to remove untracked files well? E.g. something like `$ git clean -fdx`?

Reply:
In this flow, the folder with generated documentation is considered untracked. We're only cleaning modified not-staged files here.